wazo_calld/config.py

```diff
@@ -1,15 +1,23 @@
 # Copyright 2015-2024 The Wazo Authors  (see the AUTHORS file)
 # SPDX-License-Identifier: GPL-3.0-or-later
 
+from __future__ import annotations
```

Comment:
From all our repos, on 739 import `__future__`, there are 519 imports without blank line before license comment

Reply:
Yep, I saw both forms, so I chose the one I preferred...
I find it so ugly without the space, objection to changes the 519 without spaces?

Reply:
FYI CPython adds a blank line: https://github.com/python/cpython/blob/a07cf4ce25205d836a6b30cb629eda139d2a518e/Doc/tools/extensions/c_annotations.py#L12

Reply:
no objection